sdk/search/azure-search-documents/README.md

```diff
@@ -39,12 +39,12 @@ The SDK provides two clients.
 
 #### Create a SearchServiceClient
 
-Once you have the values of the Cognitive Search Service [URL endpoint](https://docs.microsoft.com/en-us/azure/search/search-create-service-portal#get-a-key-and-url-endpoint) 
-and [admin key](https://docs.microsoft.com/en-us/azure/search/search-security-api-keys) you can create the Search Service client:
+Once you have the values of the Azure Cognitive Search service [URL endpoint](https://docs.microsoft.com/en-us/azure/search/search-create-service-portal#get-a-key-and-url-endpoint) 
```

Comment:
Remove en-us from all docs.microsoft.com links. It'll redirect users to a version according to their locale.

Reply:
Done.